### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by galantra** [theolizard/remote-numpad#1]
Since it's open source: Have you considered uploading it to the F-Droid store?
That's where I look for apps first.

**Comment by theolizard** [comment]
I did not even know alternate app stores existed! o_0
I will have a look, thanks!

**Comment by theolizard** [comment]
I have sent a registration request to F-Droid.

**Comment by IzzySoft** [comment]
Looks like [the request](https://gitlab.com/fdroid/rfp/-/issues/1257) is stuck waiting for a reply from @theolizard …

@galantra maybe you could provide the APK (ideally, attaching it to its corresponding release), so I can take the app into [my repo](https://apt.izzysoft.de/fdroid) meanwhile? Having set up its metadata there, I could in return then provide you with a PR containing the requested Fastlane structure (or at least a "minimal starter package" for that, with some hints on expanding).

**Comment by theolizard** [comment]
I have not had too much time to work on this project in the past few months. I am working on it this weekend and I will integrate Fastlane and finish the registration process for this release ;)

**Comment by theolizard** [comment]
The fastlane structure has been integrated and a tag has been created. Correct me if I am wrong but I think we just have to wait now.

**Comment by IzzySoft** [comment]
Well, yes and no. Yes, waiting is involved (and if we leave it at that, will cover 95% of it). Thanks for letting me know it's ready, so I can trigger the bot once more (done) and, if it gives OK, setup metadata – **then** it's waiting (with the current backlog, I expect some weeks at least, to be honest – we've got far too few packagers).

And no – because if you could attach a signed APK to your current (and future) release(s), I could take your app into [my repo](https://apt.izzysoft.de/fdroid) right away.

**Comment by IzzySoft** [comment]
I've just kicked the issue-bot again, so it should perform a rescan and hopefully confirm all is fine. Before that I had to slightly edit your initial post to avoid that "too-many-giturls" bot-failure. Let's see if we can get it through now: I've meanwhile advanced to "junior packager", and so far Java and Kotlin apps I seem to manage quite fine (even get a feeling for those Flutter thingies slowly); which means chances are good I can now process this myself successfully :crossed_fingers:

But first let's see what the bot reports this time.
